packages/mui-material-next/src/Divider/Divider.tsx

```diff
@@ -54,48 +55,52 @@ const DividerRoot = styled('div', {
         styles.textAlignLeft,
     ];
   },
-})(
-  ({ theme, ownerState }) => ({
-    margin: 0, // Reset browser default style.
-    flexShrink: 0,
-    borderWidth: 0,
-    borderStyle: 'solid',
-    borderColor: (theme.vars || theme).palette.divider,
-    borderBottomWidth: 'thin',
-    ...(ownerState.absolute && {
-      position: 'absolute',
-      bottom: 0,
-      left: 0,
-      width: '100%',
-    }),
-    ...(ownerState.light && {
-      borderColor: theme.vars
-        ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
-        : alpha(theme.palette.divider, 0.08),
-    }),
-    ...(ownerState.variant === 'inset' && {
-      marginLeft: 72,
-    }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'horizontal' && {
-        marginLeft: theme.spacing(2),
-        marginRight: theme.spacing(2),
+})<{ ownerState: DividerOwnerState }>(
+  ({ theme, ownerState }) => {
+    const { vars: tokens } = theme;
+
+    return {
+      margin: 0, // Reset browser default style.
+      flexShrink: 0,
+      borderWidth: 0,
+      borderStyle: 'solid',
+      borderColor: tokens.sys.color.outlineVariant,
+      borderBottomWidth: 'thin',
+      ...(ownerState.absolute && {
+        position: 'absolute',
+        bottom: 0,
+        left: 0,
+        width: '100%',
       }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'vertical' && {
-        marginTop: theme.spacing(1),
-        marginBottom: theme.spacing(1),
+      ...(ownerState.light && {
+        borderColor: theme.vars
+          ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
+          : alpha(theme.palette.divider, 0.08),
       }),
```

Comment:
How might we support the [`light` prop](https://mui.com/material-ui/api/divider/#Divider-prop-light) within Material You? Should we drop it? `outlineVariant` is already the lighter outline token.

@zanivan

Reply:
By the [MY guidelines](https://m3.material.io/components/divider/specs#34f44feb-4251-457b-9a35-6ab77fbbaf6a), by default, the divider uses the `outline-variant`—that's the `palette.neutral-variant80` for the light theme and `palette.neutral-variant30` for the dark theme. 

As I see, if we're using our `outlineVariant`, it's currently accurate with the guidelines from MY, even with it as the lightest color we have. All in all, I think we can either drop the `light` prop, or if this ends up being a problem, we could change it to be `true` as default and have a darker tone for when it's set to `false`.

Reply:
So the options would be

1. Drop the `light` prop and use `outlineVariant`. If users want to use `outline` or another color, they can customize it. We can add a `color` prop if necessary, which is more in line with other components.
2. Keep the `light` prop but change the default to `true`. The Divider would then use `outlineVariant` by default and `outline` when `light={false}`. The default change would be a breaking change.
3. Keep the `light` prop with the same default. The Divider would then use `outline` by default and `outlineVariant` when `light={true}`. This would mean the Divider won't follow the MY guideline by default.

I lean towards 1. because:
- Using customization and/or a `color` prop makes more sense with the rest of the components and is more flexible.
- Option 2. is a middle ground, but the default being `true` would be a weird API. Also, it's still a breaking change as users would need to address if they're using the `light` prop to differentiate two styles of Dividers.
- Option 3. is not desirable, as we should follow the MY guidelines by default as much as possible.

What do you think @mnajdova?

Reply:
Hey @Best-Sardar, let's go with option 1: Drop the `light` prop. We need to add this to the `migration.md` file. We need to explain to users how to achieve the same through style overrides or the sx prop in the migration file. As a guideline, you can follow: https://github.com/mui/material-ui/blob/master/packages/mui-material-next/migration.md#chip

In this case, I would have an example with the [sx prop](https://mui.com/material-ui/customization/how-to-customize/#the-sx-prop).

Let me know if you need any help with that